docs/tech_stack.md

```diff
@@ -13,14 +13,16 @@ technology stack for compliant IoT devices.
 * JSON encoding following the core schema definition, specifying the semantic structure of the data.
 * Passes the [DAQ Validation Tool](validator.md) for all requirements.
 
-# MQTT Topic Table
-
-| Type     | Category | subFolder |                MQTT Topic              |  Schema File  |
-|----------|----------|-----------|----------------------------------------|---------------|
-| state    | state    | _n/a_     | `/devices/{device_id}/state`           | state.json    |
-| config   | config   | _n/a_     | `/devices/{device_id}/config`          | config.json   |
-| pointset | event    | pointset  | `/devices/{device_id}/events/pointset` | pointset.json |
-| system   | event    | system    | `/devices/{device_id}/events/system`   | system.json   |
+# MQTT Topic Suffix Table
+
+| Type     | Category | subFolder | MQTT Topic Suffix   | Schema File   |
+| -------- | -------- | --------- | ------------------- | ------------- |
+| state    | state    | _n/a_     | `#/state`           | state.json    |
```

Comment:
I'd request not to use # as the prefix since it's also the wildcard, and it's not quite the same.  I'd write this as {topic_prefix}/state, and then it's easy to specify "topic_prefix" somewhere and it's a (conceptual) variable substitution!